content/en/users/workload-manager/_index.md

```diff
@@ -4,9 +4,463 @@ linkTitle: "Workload Manager"
+The pilot-based scheduling feature solves many problems of using
+heterogeneity and unstable distributed computing resources.
+* Includes easy extensions to customize the environment checks to address the
+needs of a particular community.
+Users can choose appropriately computing and storage resources maximising their
+usage efficiency for particular user requirements.
+* Handles different storage supporting both cloud and grid capacity.
+* Provides a user-friendly interface that allows users to choose among different
+DIRAC services.
+
+## Target User Groups
+
+The service suits for the established Virtual Organization communities, long
+tail of users, SMEs and Industry
+
+* EGI and EGI Federation participants
+* Research communities
+
+## Architecture
+
+The EGI Workload Manager service (DIRAC4EGI) is a cluster of DIRAC services
+running on EGI resources (HTC, CLOUD, HPC) supporting multi-VO.  All the DIRAC
+services are at or above TRL8.  The main service components include:
+
+* Workload Management System (WMS) architecture is composed of multiple loosely
+coupled components working together in a collaborative manner with the help of
+a common Configuration Services ensuring reliable service discovery
+functionality. The modular architecture allows to easily incorporate new types
+of computing resources as well as new task scheduling algorithms in response to
+evolving user requirements. DIRAC services can run on multiple geographically
+distributed servers which increases the overall reliability and excellent
+scalability properties.
+* REST server providing language neutral interface to DIRAC service.
+* Web portal provides simple and intuitive access to most of the DIRAC
+functionalities including management of computing tasks and distributed data.
+It also has a modular architecture designed specifically to allow easy extension
+for the needs of particular applications.
+
+![Fig. 1 - The DIRAC Web portal](./figure_1.png)
+
+Fig. 1 - The DIRAC Web portal
+
+## How to access the EGI Workload Manager service
+
+There are several options to access the service:
+
+1. Members of a scientific community whose resources pool is already configured
+   in the DIRAC4EGI instance > can use the
+   [DIRAC4EGI web portal](https://dirac.egi.eu:9443/DIRAC/)
+   to access the service, or use
+   [DIRAC Client](https://wiki.egi.eu/wiki/HOWTO22#Using_DIRAC_through_the_client).
+1. Individual researchers who want to do some number crunching for a limited
+   period of time, with a reasonable (not too high) number of CPUs > can use
+   the catch-all VO resource pool (vo.access.egi.eu). Submit a request for this
+   (CST to check identity and justification of use).
+1. Representatives of a community who want to try DIRAC and EGI > Same as #2.
+1. Representative of a community who wants to request DIRAC for the community's
+   own resource pool > Submit a request via the Marketplace, CST to call back
```

Comment:
CST again

Reply:
As before, "Submit a request through the EGI Marketplace selecting Compute > Workload Manager from the top menu"

Reply:
98e1ece77c4e9d92360ecf751b10879aca046b2b